tests/test_consumer.py

```diff
@@ -692,7 +692,7 @@ def test_full_consume_compose_msg(mock_get_session):
         'note': '',
         'data': {
             'productmd.compose.id': 'RHEL-X.0-20180101.1',
-            'type': 'compose',
+            'type': 'productmd-compose',
```

Comment:
Hmm, will this break Greenwave though?

Ah, I guess not. For composes, Greenwave doesn't care about `type` at all, it just looks for `productmd.compose.id`. Cool.